task_1/index.html

```diff
@@ -1,45 +1,110 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Task 1</title>
     <link rel="stylesheet" href="./style.css">
 </head>
-
 <body>
     <div class="wrapper">
         <div class="header">
+            <ul class="nav-list">
+                <li id="store-image" onclick="onclick_navbar_item('store-image')"><img class="store-size" src="./Assets/images/coffee-icon-png-13667.png"></li>
+                <li id="home" onclick="onclick_navbar_item('home')" class="hover-cursor">Home</li>
+                <li id="about-us" onclick="onclick_navbar_item('about-us')" class="hover-cursor">About Us</li>
+                <li id="our-lands" onclick="onclick_navbar_item('our-lands')" class="hover-cursor">Our Lands</li>
+                <li id="our-suppliers" onclick="onclick_navbar_item('our-suppliers')" class="hover-cursor">Our Suppliers</li>
+                <li id="products" onclick="onclick_navbar_item('products')" class="hover-cursor">Products</li>
+                <li id="contact-us" onclick="onclick_navbar_item('contact-us')" class="hover-cursor">Contact Us</li>
+                <li id="profile-image" onclick="onclick_navbar_item('profile-image')"><img class="profile-size" src="./Assets/images/profile Image.png"></li>
```

Comment:
Use kebab case for the all image, class, ids to be consistent  
```suggestion
                <li id="profile-image" onclick="onclick_navbar_item('profile-image')"><img class="profile-size" src="./Assets/images/profile-image.png"></li>
```